perceval/rendering/pdisplay.py

```diff
@@ -258,6 +342,22 @@ def _pdisplay(bsc, **kwargs):
     return pdisplay_state_distrib(bsc, **kwargs)
 
 
+@dispatch(float)
+def _pdisplay(f, **kwargs):
+    opts_simple = {}
+    if "precision" in kwargs:
+        opts_simple["precision"] = kwargs["precision"]
+    return simple_float(f, **opts_simple)[1]
```

Comment:
I'm pretty sure it would be better to pass `**kwargs` to `simple_float` than build a `opts_simple` dictionary. Same in the complex signature.

Reply:
implement a fix